sdk/wren-langchain/README.md

```
@@ -26,6 +26,25 @@ Complete runnable demos:
   conditional edges). Use this when you need custom routing, state, or
   streaming.
 
+### Routing through OrcaRouter
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)